assets/css/main.css

```diff
@@ -1,624 +1,693 @@
+/*
+Red/Teal
+rgba(204, 40, 41, 1), #4ba58c
+*/
+/*
+Grey-Blue
+
+*/
 html, body {
   margin: 0;
   padding: 0;
   height: 100%;
-  font-family: helvetica, sans-serif;
+  font-family: 	"mr-eaves-xl-modern", helvetica;
+  font-weight:400;
   line-height: 1.3;
-  color: #2b2b2b;
+  color:rgba(37,52,62, 1);
```

Comment:
If you wont ever use opacity, but you want to use rgb you can just use `rgb(37,52,62)` (any benefit to using rgb over hex?)